Sprint-3/quote-generator/quoteGeneratorApp.html

```diff
@@ -3,13 +3,15 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
     <title>Title here</title>
```

Comment:
This needed to be changed to quote generator

Reply:
I have changed